app/main.py

```diff
@@ -2,0 +1,26 @@
+import os
+
+
+def move_file(command: str) -> None:
+    command = command.split()
+    route = command[2].split("/")
+    if command[0] != "mv":
+        return
+    if len(route) > 1:
+        route.pop()
+    else:
+        new_file_name = route[0]
+        with (
+            open(command[1], "r") as file_in,
+            open(new_file_name, "x") as file_out,
+        ):
+            file_out.write(file_in.read())
+        os.remove(command[1])
+        return
+    for directory in route:
+        if not os.path.exists(directory):
+            os.mkdir(directory)
```

Comment:
Code Efficiency: You are creating a directory if it does not exist. However, if the directory already exists, this will lead to an error. Consider checking if the directory does not exist before trying to create it.